paxml/learners.py

```diff
@@ -335,7 +369,15 @@ def update_states(
     """
     p = self._hparams
 
-    grads, valid_step = self.scale_gradients(grads)
+    grad_norm, valid_step = self.get_grad_norm_valid_step(grads)
+
+    using_ga = hasattr(p.optimizer, 'num_sub_batches')
```

Comment:
nit: let's use `using_grad_accum`

most readers might not know what `ga` means